From 58d698185e92143454a1628046d53a8db1769df9 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Tue, 28 Nov 2017 17:33:19 +0900
Subject: [PATCH 380/909] media: i2c: adv748x: Delete height calculation at
 interlacd mode

commit fbc325b96245c8009b969dce1ceea70b52c2e3fe from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

In the interlaced mode, the HDMI driver of the ADV7482 does not halve
the height. Remove 1/2 calculation so as to process same as HDMI driver.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/i2c/adv748x/adv748x-afe.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/media/i2c/adv748x/adv748x-afe.c b/drivers/media/i2c/adv748x/adv748x-afe.c
index be0a14bc9d93..4f432164f4d9 100644
--- a/drivers/media/i2c/adv748x/adv748x-afe.c
+++ b/drivers/media/i2c/adv748x/adv748x-afe.c
@@ -124,9 +124,6 @@ static void adv748x_afe_fill_format(struct adv748x_afe *afe,
 
 	fmt->width = 720;
 	fmt->height = afe->curr_norm & V4L2_STD_525_60 ? 480 : 576;
-
-	/* Field height */
-	fmt->height /= 2;
 }
 
 static int adv748x_afe_std(v4l2_std_id std)
-- 
2.17.1

